util/expensivequery/expensivequery.go

```diff
@@ -47,7 +53,8 @@ func (eqh *Handle) Run(sm util.SessionManager) {
 	for {
 		select {
 		case <-ticker.C:
-			if log.GetLevel() > zapcore.WarnLevel {
+			eqh.checkAll()
```

Comment:
Should we use an individual ticker here, for 2 reasons:
1. check the expensive queries every 1 second costs too much
2. maybe we need to use millisecond as the unit for max_execution_time